plugin/trino-phoenix/src/main/java/io/trino/plugin/phoenix/PhoenixConfig.java

```diff
@@ -16,17 +16,35 @@
 import com.google.common.base.Splitter;
 import com.google.common.collect.ImmutableList;
 import io.airlift.configuration.Config;
+import io.airlift.configuration.ConfigDescription;
 import io.airlift.configuration.validation.FileExists;
 
+import javax.validation.constraints.Max;
+import javax.validation.constraints.Min;
 import javax.validation.constraints.NotNull;
 
 import java.util.List;
 
 public class PhoenixConfig
 {
+    /*
+     * Normally this should be kept between 1 and 70 or so (with HBase and Phoenix defaults).
+     * A large value here makes sense when the Guidepost-width in Phoenix has been reduced.
```

Comment:
IMO remove this comment and instead add `A large value here makes sense when the Guidepost-width in Phoenix has been reduced.` to the comment for `maxScansPerSplit`.